api/src/main/java/org/pmiops/workbench/db/dao/RdrExportDao.java

```diff
@@ -6,15 +6,20 @@
 import org.pmiops.workbench.db.model.DbRdrExport;
 import org.springframework.data.jpa.repository.Query;
 import org.springframework.data.repository.CrudRepository;
+import org.springframework.data.repository.query.Param;
 
 public interface RdrExportDao extends CrudRepository<DbRdrExport, Long> {
 
-  @Query(
-      nativeQuery = true,
-      value =
-          "select u.user_id from user u LEFT JOIN rdr_export rdr on"
-              + " u.user_id = rdr.export_id and rdr.entity_type = 1 where "
-              + "u.last_modified_time > rdr.last_Export_date or rdr.export_id is null")
+  public final String workspaceIdsToExportQuery =
+      "select w.workspace_id from  workspace w LEFT JOIN "
+          + "rdr_export rdr on w.workspace_id = rdr.entity_id and rdr.entity_type = 2 where "
+          + "w.last_modified_time > rdr.last_export_date or rdr.entity_id IS NULL";
+  public final String unchangedWorkspaceIdsQuery =
```

Comment:
Do we need to include ALL workspaces? Seems like perhaps we only actually need to include all workspaces that we've exported before (maybe minus any that have been modified since last export). If so - this would want to compare against the RDR export table instead of the workspace table.

There may be some workspaces that were deleted, or predate the RDR export workflow that perhaps are not found in that table.